builtin-functions/kphp-light/stdlib/confdata-functions.txt

```diff
@@ -1,4 +1,6 @@
-<?
+<?php
+
+// ===== SUPPORTED =====
```

Comment:
What for?
Isn't it supported by default, unless the opposite is written?

Reply:
fix